src/sentry/utils/performance_issues/base.py

```diff
@@ -157,6 +157,10 @@ def is_creation_allowed_for_organization(self, organization: Organization) -> bo
     def is_creation_allowed_for_project(self, project: Project) -> bool:
         return False  # Creation is off by default. Ideally, it should auto-generate the project option name, and check its value
 
+    @classmethod
+    def is_enabled(cls) -> bool:
```

Comment:
nit: should we go with `is_detection_allowed_for_project` just cause the other class function name is already confusing enough?

Reply:
is_detection_allowed_for_project